docs/data/date-pickers/date-field/DateFieldValidation.js

```diff
@@ -1,35 +1,40 @@
 import * as React from 'react';
-import addDays from 'date-fns/addDays';
-import isWeekend from 'date-fns/isWeekend';
+import dayjs from 'dayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers';
-import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
 import Stack from '@mui/material/Stack';
 
-const today = new Date();
+const today = dayjs();
+
+const isWeekend = (date) => {
```

Comment:
It seems strange that `dayjs` does not seem to have a utility for this 🤔

Reply:
I did not find any a few weeks ago when I created this small function for another demo